package.json

```diff
@@ -1,32 +1,65 @@
 {
-  "name": "mento-sdk",
-  "version": "1.0.0",
-  "description": "The SDK for Mento Protocol",
-  "main": "index.js",
+  "name": "@mentoprotocol/sdk",
+  "version": "0.1.0",
+  "license": "MIT",
+  "author": "Mento Labs",
+  "main": "dist/index.js",
+  "module": "dist/sdk.esm.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "src"
```

Comment:
src should not be included. This specifies which files should be included in the bundle published to npm